kolibri/core/assets/src/vue/session-nav-widget/index.vue

```diff
@@ -79,10 +104,7 @@
         return this.fullname;
       },
       userkind() {
-        if (this.deviceOwner) {
-          return '';
-        }
-        return this.kind;
+        return this.kind[0];
```

Comment:
Simplified this because I wasn't sure about the level of complexity we wanted.
@indirectlylit you mentioned that you'd like it to display `SUPERUSER`, and it does now, but I also noticed that there is a settings here that  `Device Owner` takes the place of a name for `SUPERUSER`s, and I think that's likely the reason that they removed this tag originally.

Also, we discussed how looking through each role and deciding which was the most powerful is probably the "smartest" way to go. Do you want me to leave it this way or add those checks?